listeners/views/daily-checkins.js

```diff
@@ -0,0 +1,19 @@
+const { dailyCheckinsFeedback } = require('../../services/aws/index');
+
+// eslint-disable-next-line consistent-return
+const postDailychechinFeedback = async ({ ack, body }) => {
+  await ack({
+    response_action: 'clear',
+  });
+  try {
+    const feedbackData = Object.values(body.view.state.values).map((value) => ({
+      question: Object.keys(value)[0],
```

Comment:
The question should be the question, it is static select id right now.

Reply:
done